docs/blogs/WhyVulnerableApp.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Why VulnerableApp
```

Comment:
Title and file name can be changed to something else ... may be `Overview of Owasp-VulnerableApp` or if you have any other idea.

 @hemantgs  any suggestions ?

Reply:
@hexxdump  also please add the link of medium article for now to Read me ... in the last. 
Create a new Heading named Blogs and then link.

Reply:
@hexxdump thanks for adding the links also can you please rename the file and also the title as mentioned above.

Reply:
done, pls review